filament/src/details/Engine.cpp

```diff
@@ -130,6 +135,27 @@ void FEngine::createAsync(CreateCallback callback, void* user,
     callbackThread.detach();
 }
 
+Engine::Config FEngine::validateConfig(const Config* config) noexcept
+{
+    constexpr uint32_t ARENA_COMMANDS_DELTA = 1;    // Rule of thumb: perRenderPassArenaMB must be roughly 1 MB larger than perFrameCommandsMB
+    constexpr uint32_t NUM_FRAMES_OVERLAP = 3;      // Number of potential concurrent frames in flight
+
+    Config validConfig;
+    if (config == nullptr)
+    {
```

Comment:
```suggestion
    if (!config) {
```

Reply:
Yeah, that's my C# showing through :) .  Most of my career has been spent in C++, but recent years have focused on C#.